laa_court_data_api_app/routers/ping.py

```diff
@@ -10,7 +10,7 @@
 
 @router.get('/ping', response_model=Ping)
 async def ping(settings: AppSettings = Depends(get_app_settings)):
-    logger.debug('Calling GET Endpoint')
+    logger.info('Calling GET Endpoint')
```

Comment:
The ping endpoint is now used for readiness and live probe. We can probably remove this log message to avoid flooding the logs.